pkg/container-hook/tracer.go

```diff
@@ -73,6 +73,16 @@ const (
 	defaultContainerCheckInterval  = 10 * time.Second
 )
 
+const (
+	// config.json is typically less than 100 KiB.
+	// 16 MiB should be enough.
+	configJsonMaxSize = int64(16 * 1024 * 1024)
+
+	// pid files store a string with a int32 value, so 11 characters.
+	// Keep a larger buffer to be able to notice errors with strconv.Atoi.
+	pidFileMaxSize = int64(32)
```

Comment:
We use `strconv.Atoi`, which returns an `int`. And `int` seems to be 64 bits because it can return 19 characters (see example in https://go.dev/play/p/lYUcxP300Vc). We then check the conversion to 32 bits. So I prefer to keep a buffer large enough for that, to make sure that `strconv.Atoi` would return a problem if it is larger than 19.